cli/src/cli.rs

```diff
@@ -2648,6 +2684,13 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
                         .takes_value(true)
                         .help("Write the account data to this file"),
                 )
+                .arg(
+                    Arg::with_name("account_data_output")
+                        .long("account-data-output")
```

Comment:
What if we just called this format:
```suggestion
                    Arg::with_name("format")
```

Reply:
A `.default_value("..")` would be nice here too

Reply:
@mvines Well, there already exists `--output`, so I wanted to avoid too similarly named argument..